app/models/sam_account_reckoner.rb

```diff
@@ -24,8 +26,12 @@ def should_clear?
     user.persisted? && user.duns_number_changed?
   end
 
-  def user_in_sam?
-    client.duns_is_in_sam?(duns: user.duns_number)
+  def sam_status
+    if client.duns_is_in_sam?(duns: user.duns_number)
```

Comment:
updated to use symbols instead of integers at 8cbae1d

better?

I think the only place we need to ref integers rather than symbols is in AR queries

eg

```ruby
User.where(sam_status: :sam_pending)
```

does not work. So you must do:

```ruby
User.where(sam_status: 0)
```

this will be fixed in Rails 5.  https://github.com/rails/rails/issues/19964#issuecomment-98591859